ui/src/pages/Questions/EditAnswer/index.scss

```diff
@@ -8,6 +8,7 @@
       height: calc(100% - 12px);
       overflow-y: scroll;
       z-index: 1;
+      background-color: #eee;
```

Comment:
This change is not particularly necessary. Also not particularly appropriate - the `.content` node has a `.bg-white` className, which would have to be cleaned up first if changed.

Suggest cancelling the changes to this file and I'll merge #428.

Thanks!

Reply:
This is currently being fixed in #423.